keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandlerTests.kt

```diff
@@ -106,7 +106,7 @@ internal class ApplicationLoadBalancerHandlerTests : JUnit5Minutests {
   private val sub2 = Subnet("subnet-1", vpc.id, vpc.account, vpc.region, "${vpc.region}d", "internal (vpc0)")
   private val sg1 = SecurityGroupSummary("testapp-elb", "sg-55555", "vpc-1")
 
-  private val model = ApplicationLoadBalancerModel(
+  private fun model(port: Int = 7001, healthCheckPort: String = "7001") = ApplicationLoadBalancerModel(
```

Comment:
Changed this from a variable to a function to make it easier to test with different port parameters.